be/src/exec/pipeline/scan/scan_operator.h

```diff
@@ -166,8 +166,8 @@ class ScanOperatorFactory : public SourceOperatorFactory {
     Status prepare(RuntimeState* state) override;
     void close(RuntimeState* state) override;
 
-    bool need_local_shuffle() const override { return _need_local_shuffle; }
-    void set_need_local_shuffle(bool need_local_shuffle) override { _need_local_shuffle = need_local_shuffle; }
+    bool could_local_shuffle() const override { return _could_local_shuffle; }
+    void set_could_local_shuffle(bool could_local_shuffle) override { _could_local_shuffle = could_local_shuffle; }
```

Comment:
These could be removed, since they are the same as the methods of `SourceOperator`.

Reply:
Yes, exactly. Updated